uploader.json

```diff
@@ -1437,7 +1437,12 @@
             "symbolic_name": "x86_retp_enabled"
         },
         {
-            "file": "/var/log/dirsrv/.*/()*(errors|errors\\.2.*)",
+            "file": "/var/log/dirsrv/.*/access*",
```

Comment:
@wushiqinlou this should be moved to the globs section of the JSON and changed to "/var/log/dirsrv/*/access*".  You should go ahead and change the `errors` spec as well.